### ISSUE TITLE
tain transfers config problem

### ISSUE THREAD

**Issue by linlinhan-git** [pascal-lab/Tai-e#116]
### Overall Description

Hello, teacher! When will the configuration of tai-e stain transfer be released? When I saw the video yesterday, you said that there are currently thousands of stain transfer configurations, and the company has just started to involve this direction of sensitive leakage, and it is still under investigation

### Expected Behavior

publish tain transfers config

### Current Behavior

Technical research

### Tai-e Arguments

<details>
  <summary>Click here to see Tai-e Options</summary>
  

  ```yaml
  {{The content of 'output/options.yml' file}}
  ```
</details>
<details>
  <summary>Click here to see Tai-e Analysis Plan</summary>
  

  ```yaml
  {{The content of 'output/tai-e-plan.yml' file}}
  ```
</details>

### Tai-e Log

<details>
  <summary>Click here to see Tai-e Log</summary>
  

  ```
  {{The content of 'output/tai-e.log' file}}
  ```
</details>

**Comment by zhangt2333** [comment]
The taint config rules have been collected, the release is still pending as there are some remaining tasks to complete. We appreciate your patience in the meantime. 😊

**Comment by zhangt2333** [comment]
Hello @linlinhan-git,

The taint configuration rules discussed in this issue have been released in [commit 36fc4f7](https://github.com/pascal-lab/Tai-e/commit/36fc4f7ddd4e91fe304e570f8e6132a82d67ca35) and are now available in our [v0.5.1 release](https://github.com/pascal-lab/Tai-e/releases/tag/v0.5.1). You can find detailed documentation about these rules in the [Taint Analysis section](https://tai-e.pascal-lab.net/docs/0.5.1/reference/en/taint-analysis.html#pre-prepared-commonly-used-taint-configuration) of our documentation.
